components/bingx/actions/account-interface/bingx-get-positions.mjs

```diff
@@ -0,0 +1,29 @@
+import bingx from "../../bingx.app.mjs";
+
+export default {
+  name: "BingX Account GetPositions",
+  version: "0.0.1",
+  key: "bingx-account-get-positions",
+  description: "Perpetual Swap Positions [reference](https://bingx-api.github.io/docs/swap/account-api.html#_2-perpetual-swap-positions).",
+  props: {
+    bingx,
+    symbol: {
```

Comment:
Hi there. Looks like you are using the `symbol` in some files. It would be great if you could move it to the `app`

Reply:
@vellames - I think symbol need to remain in the actions. App is authenticating to exchange. But, if user is placing orders, they can do it for different symbols every time. Hence, I think symbol should remain in the actions.

Reply:
Hi there. Looks like the options are the same code, what is changing is the default value. In that case, I would suggest you to move to `app` and override the params as you wish, this way you don't need to repeat the `options` code :)

Reply:
@vellames - you mean to say, I can define the parameter in app and yet can chose them in actions when adding actions? 

Let me try to find other examples and check.

Reply:
@trendoscope-algorithms You can define a the parameter in app and override the props in each action as you wish :) In that previous example. `symbol` would "inherit" all props from `app.symbol`, but the `default` prop would be overridden. It works similar to a polimofirsm

Reply:
@vellames - got it. Thanks :) I was under the impression that adding properties in app will prompt for these parameters while authenticating the app.